recipe/meta.yaml

```diff
@@ -40,11 +40,17 @@ test:
     - quantecon.markov
     - quantecon.random
     - quantecon.util
+  requires:
+    - pip
+    - pytest
```

Comment:
Since you only call `pip check` in the tests, you could remove pytest. Or you could keep pytest and use it to run the unit tests. Something like `pytest -v  --pyargs quantecon`. Though the tests are a little slow, so it's up to you and others on the team to decide if it's worth it.

SO in short, either remove pytest or leave it and run it.

Reply:
`pandas` will be required too if you run `pytest`

Reply:
ok I'll just remove them altogether :)